### ISSUE TITLE
Squiz.Strings.EchoedStrings gives false positive when echo'ing using an inline condition

### ISSUE THREAD

**Issue by u01jmg3** [squizlabs/PHP_CodeSniffer#529]
```xml
<rule ref="Squiz.Strings.EchoedStrings" />
```

---

- The following gives `Echoed strings should not be bracketed`

```php
<?php echo ($loan_device->returndate == 0) ? 'Not returned' : date('d/m/Y', $loan_device->returndate); ?>
```

- The following solves the above (by separating out and using `$x`) but I don't why the above isn't valid

```php
<?php 
    $x = date('d/m/Y', $loan_device->returndate);
    echo ($loan_device->returndate == 0) ? 'Not returned' : $x;
?>
```

**Comment by u01jmg3** [comment]
Is this valid?

**Comment by aik099** [comment]
Sounds like a bug.

**Comment by gsherwood** [comment]
What's wrong with the super-easy-to-read:
```php
if ($loan_device->returndate == 0) {
    echo 'Not returned';
} else {
    echo date('d/m/Y', $loan_device->returndate);
}
```
?? :smile: 

Anyway, bug is fixed.

**Comment by u01jmg3** [comment]
Haha, but one liners are so much quicker to write :stuck_out_tongue_winking_eye: 

Thanks for fixing

**Comment by u01jmg3** [comment]
When will this fix be live? Currently on 2.3.0

**Comment by gsherwood** [comment]
> When will this fix be live? Currently on 2.3.0

Next release should be this week.
